quickstarts/aws/amazon-athena/config.yml

```diff
@@ -37,4 +37,9 @@ keywords:
   - amazon web services
   - database
 installPlans:
-  - aws-cloudwatch-metric-streams
\ No newline at end of file
+  - aws-cloudwatch-metric-streams
+
+dashboards:
+  - athena
```

Comment:
I think this may be an accidental duplicate!

```suggestion
```

Reply:
This is actually related to the question on whether we should include the existing athena/athena.json which is Sample Metric based vs the amazon-athena/amazon-athena.json that is migrated from infra templates and is dimensional metrics based. I'm not sure if we want to keep both? The sample metric json has more charts, the dimensional metrics only has 2. I'm not sure if it's possible to leave both so users have the choice?

Reply:
Hey @lroyce, is it possible to merge the two dashboards? That way we get the best of both worlds!

Reply:
@aswanson-nr that's definitely doable: would we put all together in a single dashboard? Or would we want a tabbed dashboard, one for metrics, one for samples?

Reply:
My understanding is that there is no difference between samples and metrics to the end user when they're looking at the dashboard. If that's correct, then let's just toss them all on one tab. If not, then separate tabs sounds great!

Reply:
@aswanson-nr and @josephgregoryii `amazon-athena.json` was merged with `athena.json` and now has both widget queries.